sympy/matrices/matrices.py

```diff
@@ -648,7 +648,7 @@ def _normalize_op_args(self, op, col, k, col1, col2, error_str="col"):
 
     def _permute_complexity_right(self, iszerofunc):
         """Permute columns with complicated elements as
-        far right as they can go.  Since the `sympy` row reduction
+        far right as they can go.  Since the ``sympy`` row reduction
```

Comment:
sympy -> SymPy

Reply:
This isn't worth a rebuild...it can be fixed maybe with a code-wide sympy->SymPy cleanup in docstrings.